src/router/index.ts

```diff
@@ -16,35 +10,62 @@ const routes = [
       {
         path: "/",
         name: "dashboard",
-        component: Dashboard,
+        component: async () => import("@/pages/Dashboard.vue"),
       },
       {
-        path: "page-2",
-        name: "side-menu-page-2",
-        component: Page2,
+        path: "/regulations",
+        component: async () => import("@/layouts/Regulations.vue"),
```

Comment:
layout ihtiyacımız var mı gerçekten?

Reply:
zorunlu değil. Regulation route'larını tek bir çatı altığında toplamak için child route'lar olarlak gösterim. Bunun için de layout zorunluydu. Her bir route child olarak değil de bağımsız bir şekilde oluşturulursa gerek olmaz. Ama şu faydası da var layotu'un. Evet çok küçük div ama layout içinde <div class="w-1/2"> diye bir şey var. Bütün regulation sayfalarında geçerli oluyor. Herbirinde tek tek yazmıyoruz. Belki ileride gerekli olabilecek şeyler de olur diye bıraktım.